frontend/public/components/catalog/_catalog.scss

```diff
@@ -161,7 +170,10 @@ $co-modal-ignore-warning-icon-width: 30px;
   }
 
   // Enable scrolling on the modal
-  &__overlay {
+  .pf-c-modal-box &__overlay {
```

Comment:
This is invalid.  None of the following CSS is doing anything.

Reply:
I'm adding this back in because removing it breaks scroll on modals with a hint block.